Added Scheduler

07d8b5 opened the pull request:
- Swapped to 480p for smaller file sizes during testing.
- Changed main.py for easier adding of subprocesses.
- Fixed some issues with paths.
- Made config/schedule.yaml
- Added scheduler.

07d8b5 committed:
Added scheduler, config yaml, restructured video, changed main for easier subprocessing.

sisandov commented:
Remove the logfile, and add a way to install all dependencies needed for the project (maybe a make target). I think a python virtual environment with a dependencies file is needed.

07d8b5 committed:
Add log_file.log to .gitignore

07d8b5 committed:
Add log_file.log to .gitignore

07d8b5 committed:
Add log_file.log to .gitignore -- fixed typo

07d8b5 committed:
Remove contents of logs and playlists.

07d8b5 committed:
Add logs and playlists to .gitignore

07d8b5 commented:
- Removed log_file.log contents
- Rebased with current development branch; now has the venv changes
- Changed upnext.txt and playlist.txt to more sane default contents
- Added log_file.log, upnext.txt, and playlist.txt to .gitignore
